native/cocos/platform/interfaces/modules/canvas/CanvasRenderingContext2D.cpp

```diff
@@ -254,10 +255,11 @@ void CanvasRenderingContext2D::setFont(const ccstd::string &font) {
             fontSizeStr = results[2].str();
             fontName = results[5].str();
         }
-        bool isItalic = font.find("italic", 0) != ccstd::string::npos;
+        bool isItalic = font.find("italic", 0) != ccstd::string::npos || font.find("Italic", 0) != ccstd::string::npos;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/font#formal_definition
Italic may be edited to ITALIC

Reply:
consider to unity font-name when refactor this part later on.